generic-helpers/src/main/java/io/elimu/a2d2/helpers/JedisHelper.java

```diff
@@ -0,0 +1,63 @@
+package io.elimu.a2d2.helpers;
+
+import java.time.Duration;
+
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+import redis.clients.jedis.JedisPoolConfig;
+import redis.clients.jedis.Protocol;
+
+public class JedisHelper {
+
+	static final int JEDIS_PORT = Integer.parseInt(System.getProperty("cache.jedis.port", "6379"));
+	static final String JEDIS_PWD = System.getProperty("cache.jedis.password");
+	static final String JEDIS_HOST = System.getProperty("cache.jedis.host", "localhost");
```

Comment:
@marianbuenosayres if someone miss to add **cache.jedis.host** property in JAVA_OPTS ? it will try to connect to localhost, instead it should throw exception for required env variable in order to connect REDIS server.

Reply:
wouldn't it be better to throw runtime exception "Required env variable not found" instead of trying to connect to localhost and getting timeout exception?

Reply:
@ddjain updated to do that just now